<ui version="4.0" >
 <class>DynamicBrushOptions</class>
 <widget class="QWidget" name="DynamicBrushOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>136</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Shape:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="comboBoxShapes" >
     <item>
      <property name="text" >
       <string>Brush</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Bristles</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Program:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="comboBoxShapePrograms" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Coloring:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="comboBoxColoring" >
     <item>
      <property name="text" >
       <string>Plain</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Uniform random</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Total random</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Program:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="comboBoxColoringPrograms" />
   </item>
   <item row="4" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
